app/lib/format-helpers.js

```diff
@@ -0,0 +1,19 @@
+const formatAddress = (address) => {
+  if (!address) {
+    return null
```

Comment:
I assume your formatAddress call handles a null as opposed to returning an empty array?